Correlation ID with Universal Login (#473)
auth0/docs-v2: main/docs/customize/login-pages/universal-login/customize-templates.mdx

| `organization.branding.colors.primary` | Primary branding color for Organization | #000000 |
| `organization.branding.colors.page_background` | Background color for Organization's login pages | #FFFFFF |

#### Tenant

| **Variable** | **Description** | **Example** |
| --- | --- | --- |
| `tenant.friendly_name` | Your tenant's display name | My Tenant |
| `tenant.support_email` | Support email address for your tenant | support@example.com |
| `tenant.support_url` | Support page URL for your tenant | `https://example.com/support` |
| `tenant.enabled_locales` | Comma-separated list of locales enabled for your tenant | `en, es` |

#### Tracking

<Warning>

> Correlation ID is currently in Early Access. By using this feature, you agree to the applicable Free Trial terms in Okta’s [Master Subscription Agreement](https://www.okta.com/legal). To learn more about Auth0's product release cycle, read [Product Release Stages](/docs/troubleshoot/product-lifecycle/product-release-stages). To participate in this program, contact [Auth0 Support](support.auth0.com) or your Technical Account Manager.

Reviewer: ```suggestion
Correlation ID is currently in Early Access. By using this feature, you agree to the applicable Free Trial terms in Okta’s [Master Subscription Agreement](https://www.okta.com/legal). To learn more about Auth0's product release cycle, read [Product Release Stages](/docs/troubleshoot/product-lifecycle/product-release-stages). To participate in this program, contact [Auth0 Support](https://support.auth0.com) or your Technical Account Manager.
```

</Warning>

| **Variable** | **Description** | **Example** |
| ----- | ----- | ---- |
| `correlation_id` | A unique identifier to track authentication events such as signup and successful login. 64 characters or less. Add `correlationId: "{{correlation_id}}"` to your template to track events through your custom login page. | `0001234` |

> <Warning>

Reviewer: back to back warnings here... can we swap to callout? or fold into table? this is minor but just a thought. feel free to ignore

| `organization.branding.colors.primary` | Primary branding color for Organization | #000000 |
| `organization.branding.colors.page_background` | Background color for Organization's login pages | #FFFFFF |

> #### Tenant

Reviewer: Should we use markdown here?

<table class="table"><thead>
<tr>
<th><strong>Variable</strong></th>
<th><strong>Description</strong></th>
<th><strong>Example</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td><code>tenant.friendly_name</code></td>
<td>Your tenant's display name</td>
<td>My Tenant</td>
</tr>
<tr>
<td><code>tenant.support_email</code></td>
<td>Support email address for your tenant</td>
<td>support@example.com</td>
</tr>
<tr>
<td><code>tenant.support_url</code></td>
<td>Support page URL for your tenant</td>
<td>`https://example.com/support`</td>
</tr>
<tr>
<td><code>tenant.enabled_locales</code></td>
<td>Comma-separated list of locales enabled for your tenant</td>
<td><code>en, es</code></td>
</tr>
</tbody>
</table>

#### Tracking

<Warning>

> Correlate events is currently in Early Access. By using this feature, you agree to the applicable Free Trial terms in Okta’s [Master Subscription Agreement](https://www.okta.com/legal). To learn more about Auth0's product release cycle, read [Product Release Stages](https://auth0.com/docs/troubleshoot/product-lifecycle/product-release-stages). To participate in this program, contact [Auth0 Support](support.auth0.com) or your Technical Account Manager.

Reviewer: ```suggestion
Correlate events is currently in Early Access. By using this feature, you agree to the applicable Free Trial terms in Okta’s [Master Subscription Agreement](https://www.okta.com/legal). To learn more about Auth0's product release cycle, read [Product Release Stages](docs/troubleshoot/product-lifecycle/product-release-stages). To participate in this program, contact [Auth0 Support](https://support.auth0.com) or your Technical Account Manager.
```

</Warning>

| **Variable** | **Description** | **Example** |
| ----- | ----- | ---- |

> | `correlation_id ` | A unique identifier to track authentication events such as signup and successful login. 64 characters or less. Add `correlationId: "{{correlation_id}}"` to your template to track events through your custom login page. | `0001234` |

Reviewer: ```suggestion
| `correlation_id` | A unique identifier to track authentication events such as signup and successful login. 64 characters or less. Add `correlationId: "{{correlation_id}}"` to your template to track events through your custom login page. | `0001234` |
```

<th><strong>Example</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td><code>tenant.friendly_name</code></td>
<td>Your tenant's display name</td>
<td>My Tenant</td>
</tr>
<tr>
<td><code>tenant.support_email</code></td>
<td>Support email address for your tenant</td>
<td>support@example.com</td>
</tr>
<tr>
<td><code>tenant.support_url</code></td>
<td>Support page URL for your tenant</td>
<td>`https://example.com/support`</td>
</tr>
<tr>
<td><code>tenant.enabled_locales</code></td>
<td>Comma-separated list of locales enabled for your tenant</td>
<td><code>en, es</code></td>
</tr>
</tbody>
</table>

#### Tracking

<Warning>
Correlate events is currently in Limited Early Access. By using this feature, you agree to the applicable Free Trial terms in Okta’s [Master Subscription Agreement](https://www.okta.com/legal). To learn more about Auth0's product release cycle, read [Product Release Stages](https://auth0.com/docs/troubleshoot/product-lifecycle/product-release-stages). To participate in this program, contact [Auth0 Support](support.auth0.com) or your Technical Account Manager.
</Warning>

| **Variable** | **Description** | **Example** |
| ----- | ----- | ---- |

> | `correlation_id ` | A unique identifier to track authentication events such as sign-up and successful login. 64 characters or less. Add `correlationId: "{{correlation_id}}"` to your template to track events through your custom login page. | `0001234` |

Reviewer 2: ```suggestion
| `correlation_id ` | A unique identifier to track authentication events such as signup and successful login. 64 characters or less. Add `correlationId: "{{correlation_id}}"` to your template to track events through your custom login page. | `0001234` |
```
